src/transport_performance/gtfs/cleaners.py

```diff
@@ -45,6 +46,12 @@ def drop_trips(gtfs, trip_id: Union[str, list, np.ndarray]) -> None:
         exp_type=str,
     )
 
+    # warn users if passed one of the passed trip_id's is not present in the
+    # GTFS.
+    for _id in trip_id:
+        if _id not in gtfs.feed.trips.trip_id.unique():
+            warnings.warn(UserWarning(f"trip_id '{_id}' not found in GTFS"))
+
```

Comment:
This relates to #180 :
"6. drop_trip() works with trip_ids that aren't in stop_times. This is OK since no cleaning is done, but it's slightly opaque for public function, maybe we can protect against this? Perhaps raise a warning to say not cleaning since no warnings/no valid trip_ids?"

* Can you tidy up the comment?
* This checks [ID] or IDs against trips.trip_ids. Though the initial ask was to check against stop_times. Check with @ethan-moss as raised the initial issue.
* Should we raise if no valid IDs are passed? If there’s at least one found ID, then warn if user passes IDs not present in trips / stop_times. But if there are no found IDs in trips/stop_times then raise.
* Could use a test in test_cleaners as this is a public function

Reply:
I have confirmed this morning that `gtfs_kit.drop_zombies` is used and handles unmatched trip_ids left hanging in stop_times once `drop_trips` has removed those trip_ids from the trips table.